test/claimsTests/enrollmentClaimsTests.ts

```diff
@@ -100,7 +105,17 @@ export function enrollmentClaimsTests() {
   });
 
   test("enrollment by issuer of type DID", async () => {
-    await subjectIam.createClaimRequest({
+    await enrolAndIssue({ subjectDID: userDID, requesterDID: userDID });
+  });
+
+  test("asset enrollment by issuer of type DID", async () => {
+    const assetAddress = await userIam.registerAsset();
+    const assetDID = `did:${Methods.Erc1056}:${assetAddress}`;
+    await enrolAndIssue({ subjectDID: assetDID, requesterDID: userDID });
+  });
+
+  async function enrolAndIssue({ subjectDID, requesterDID }: { subjectDID: string, requesterDID: string }) {
```

Comment:
I suggest to remove `requesterDID` since only user request enrollment

Reply:
Thanks, great point